bot/exts/summer_aoc.py

```diff
@@ -63,8 +64,9 @@ def __init__(self, bot: SirRobin):
         self.wait_task: asyncio.Task | None = None
         self.loop_task: tasks.Loop | None = None
 
-        self.is_running = False
+        self.is_running: bool = False
```

Comment:
Does this need an annotation? It should be inferable.

Reply:
I was writing this code with pyright strict mode enabled. I can remove this if that's necessary.